### ISSUE TITLE
Screenshot dragging operation never ends, if you have NoScript installed

### ISSUE THREAD

**Issue by dholbert** [mozilla-services/screenshots#2964]
STR:
 1. [Install NoScript](https://noscript.net/getit) in Firefox Nightly (and restart browser to complete installation).
 2. Click the screenshot icon on Firefox toolbar, click through the tutorial (if present), and then try to drag to create a screenshot as directed.

ACTUAL RESULTS:
It almost works, until the very end -- your dragging operation (to create a rectangle) never ends!! When you release the mouse button, the screenshot tool doesn't seem to notice. It just keeps resizing its screenshot-rect as if you're still trying to choose the region to screenshot.

(If I click wildly, then sometimes I can get it to stop, but only if I'm lucky. And then I usually end up accidentally dismissing the screenshot area from an extra click.)

**Comment by dholbert** [comment]
This happens even if you choose "Allow Scripts Globally".  So I think this is coming from one of NoScript's other security features. I do see stuff like this in the error console, after performing the STR:
> [NoScript ClearClick] Swallowed event mouseup on DIV/-1 at moz-extension://7e5216f3-903c-4cb7-9bb7-8dacb0a0be36/blank.html

**Comment by dholbert** [comment]
Yeah, looks like this is due to a bad interaction with NoScript "ClearClick" feature (which is a clickjacking protection I think).

If you disable that (via NoScript options "Advanced" tab, "ClearClick" subtab, & uncheck all boxes there), and restart Firefox, then the problem goes away and the feature works as expected.

**Comment by 6a68** [comment]
\o/ Thanks for the sleuthing! We've been talking about an FAQ document for Screenshots, and this would be a great addition. cc @johngruen

**Comment by 6a68** [comment]
I bet NoScript is disabling the click because the origin of the Screenshots script (`moz-extension://blah`) doesn't match that of the underlying page

**Comment by hackademix** [comment]
Investigating, looking for a work-around...

**Comment by SXZ1** [comment]
I didn't realize this problem had been brought up on github as well. I created a thread on the official forum and Noscript users gave me a solution long time ago:
https://forums.informaction.com/viewtopic.php?p=88474&sid=9831800ff4bc0dcb83a978810c47b147#p88474
